salt/engines/slack.py

```diff
@@ -224,5 +224,6 @@ def fire(tag, msg):
                     else:
                         # Fire event to event bus
                         fire('{0}/{1}'.format(tag, _m['type']), _m)
+            time.sleep(1)
```

Comment:
That's weird, should be ok. "while True:" is at position 9, so "time.sleep(1)" should be at position 13, same level as the for-loop.
Maybe it's not all spaces as lineout characters. I re-indented the time.sleep(1) and checked it again for weird whitespaces. Should be fine now.